.eslintrc.js

```diff
@@ -24,6 +24,8 @@ module.exports = {
     '@typescript-eslint/explicit-member-accessibility': ['off'],
     '@typescript-eslint/no-parameter-properties': ['off'],
     '@typescript-eslint/unified-signatures': ['off'],
+    'vue/v-bind-style': ['error', 'shorthand', { 'sameNameShorthand': 'always' }],
+    "vue/attribute-hyphenation": ['off'],
```

Comment:
Add a comment here why you disabled it.

Reply:
Actually, as I understand it, it causes only an IDE error, while ESLint check in CLI handles it properly. I would say that a slight inconvenience in IDE has less priority and it more local in scope. So, please revert and use `:hash-type` in actual code.